pandas/core/tools/datetimes.py

```diff
@@ -765,8 +765,8 @@ def to_datetime(
     unit : str, default 'ns'
         The unit of the arg (D,s,ms,us,ns) denote the unit, which is an
         integer or float number. This will be based off the origin.
-        Example, with ``unit='ms'`` and ``origin='unix'`` (the default), this
```

Comment:
I am pretty sure this is only referencing origin and not unit

Reply:
Yeah, it could be interpreted either way. I suggest that it's at least potentially confusing and a maintenance risk, so we'd be better off without it.

Reply:
could change the order to put the "origin=unix (the default)" before "unit=ms"

Reply:
Indeed, but I suggest that documenting what value is the default in multiple places is asking for trouble in the long run.